frontend/src/views/Layout/MainLayout.vue

```diff
@@ -1,95 +1,105 @@
-            <v-list-item
-              :to="`/${item.path}`"
-              v-else-if="item.meta.menuItem"
-              :key="`no-nested-${index}`"
-              :active-class="getActiveClass(item.path)"
-            >
-              <v-list-item-icon>
-                <v-icon v-text="item.meta.menuIcon"></v-icon>
-              </v-list-item-icon>
-              <v-list-item-content>
-                <v-list-item-title>{{ item.meta.menuName }}</v-list-item-title>
-              </v-list-item-content>
-            </v-list-item>
-          </template>
-          <v-divider></v-divider>
-          <v-list-item @click="signOut" v-if="currentUser">
-            <v-list-item-action>
-              <v-icon left>fas fa-sign-out-alt</v-icon>
-            </v-list-item-action>
-            <v-list-item-content>SignOut</v-list-item-content>
           </v-list-item>
-        </v-list>
-      </v-navigation-drawer>
+        </template>
+        <v-divider></v-divider>
+        <v-list-item @click="signOut" v-if="currentUser">
+          <v-list-item-action>
+            <v-icon left>fas fa-sign-out-alt</v-icon>
+          </v-list-item-action>
+          <v-list-item-content>SignOut</v-list-item-content>
+        </v-list-item>
+      </v-list>
+    </v-navigation-drawer>
 
-      <v-app-bar :clipped-left="$vuetify.breakpoint.lgAndUp" fixed app>
-        <v-app-bar-nav-icon
-          @click="sidebar = !sidebar"
-          v-if="currentUser"
-        ></v-app-bar-nav-icon>
-        <v-toolbar-title>
-          <router-link to="/" tag="span" style="cursor: pointer">{{
-            appTitle
-          }}</router-link>
-        </v-toolbar-title>
-      </v-app-bar>
-    </template>
-    <template slot="aboveRoute">
-      <slot name="aboveRoute"></slot>
-    </template>
-  </base-layout>
+    <v-app-bar :clipped-left="$vuetify.breakpoint.lgAndUp" fixed app>
+      <v-app-bar-nav-icon
+        @click="sidebar = !sidebar"
+        v-if="currentUser"
+      ></v-app-bar-nav-icon>
+      <v-toolbar-title>
+        <router-link to="/" tag="span" style="cursor: pointer">{{
+          appTitle
+        }}</router-link>
+      </v-toolbar-title>
+    </v-app-bar>
+    <v-content>
+      <v-container fluid>
+        <v-breadcrumbs
+          v-if="breadcrumbs.length"
+          :items="breadcrumbs"
+          divider=">"
+        ></v-breadcrumbs>
+        <router-view></router-view>
+      </v-container>
+    </v-content>
+  </v-app>
 </template>
 
 <script lang="ts">
-import { Component, Vue } from "vue-property-decorator";
+import { Component, Vue, Watch } from "vue-property-decorator";
+import _ from "lodash";
 import { mapState, mapActions } from "vuex";
 import { RouteConfig } from "vue-router";
+import firebase from "firebase/app";
 import { getMenuItems } from "@/router";
-import BaseLayout from "./BaseLayout.vue";
 import MenuLinks from "@/components/MenuLinks";
 
+const pathMap = {
+  cr: "CR",
+  sqr: "SQR",
+  te: "TE",
+  allot: "Allotment"
+};
+
```

Comment:
I think this knowledge is not relevant here and could be extracted to the routes or to the components. Please consider refactoring this.